powermon/dto/commandDTO.py

```diff
@@ -1,6 +1,7 @@
 from pydantic import BaseModel
 
 
+# TODO: update
```

Comment:
What would you like updated on the DTOs?

Reply:
its just a reminder that they need to be updated to cater for the changes that are being made